package.json

```diff
@@ -24,8 +24,7 @@
     "node-fetch": "^2.6.1",
     "protobufjs": "^6.9.0",
     "retry-request": "^4.0.0",
-    "semver": "^6.0.0",
-    "walkdir": "^0.4.0"
+    "semver": "^6.0.0"
```

Comment:
Would be nice to add an `npm run` script here to generate the list.  Also wondering - maybe document _when_ someone would need to do that.

Reply:
I will add the script. I was thinking of making it a part of `prepare` script but decided not to, at least for now.

Reply:
Done!  To answer the "when" part, the right answer is probably "never" because this code is not needed anymore in libraries. We don't load protos :)  We load JSONs for more than a year already. I'm not confident that it's actually not used, so I don't remove it, but probably with the next major version I will do that.